Contest_API.md

```diff
@@ -1058,6 +1056,37 @@ clients with `admin` or `analyst` role. The `reaction` attribute
 is available to clients with `public` role only when the contest is
 not frozen.
 
+#### POST submissions
+
+To add submissions one can use the `POST` method on the submissions endpoint.
+The `POST` must include a valid JSON object with the following attributes:
+
+| Name          | Type             | Required? | Nullable? | Description                                                                                        |
+| ------------- | ---------------- | --------- | --------- | -------------------------------------------------------------------------------------------------- |
+| language\_id  | ID               | yes       | no        | identifier of the [ language](#languages) to submit for                                            |
+| problem\_id   | ID               | yes       | no        | identifier of the [ problem](#problems) to submit for                                              |
+| team\_id      | ID               | yes       | no        | identifier of the [ team](#teams) that made the submission                                         |
+| time          | TIME             | no        | yes       | timestamp of when the submission was made                                                          |
+| entry\_point  | string           | no        | yes       | code entry point for specific languages                                                            |
+| files         | array of ARCHIVE | yes       | no        | submission files, contained at the root of the archive. Only allowed mime type is application/zip. |
```

Comment:
Why `array of ARCHIVE` instead of just `ARCHIVE`, since they are already zipped. What's the difference in behavior.

Reply:
In GET it is also an array while only one item is allowed so it is for consistency.
I think it is an array because it is an array in other places. But I’m not sure and have already said I don’t like it being an array

Reply:
It was a design choice that for consistency a file-type attribute always returns an `array of ARCHIVE`. We've debated that pretty extensively, but the details of the pro/con arguments might be lost. We could rethink that (and this time record the arguments). And indeed, I think for consistency sake we should have the `POST` should use the same structure as the `GET`.

Reply:
OK, consistency is a fine argument. In that case just explicitly mention here that the array must have length 1.

Reply:
Here and in the GET 🙂